tests/execute/reboot/main.fmf

```diff
@@ -4,6 +4,10 @@ duration: 20m
     summary: Verify that reboot during test works
     test: ./basic.sh
 
+/efi:
+    summary: Verify that reboot during test with and without efi option works
+    test: ./efi.sh
+    enabled: false
```

Comment:
Perhaps a short comment explaining why the test is disabled would be useful.

Reply:
Makes sense. I've added some comments in my latest commit.